app/Http/Controllers/TransactionController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Inertia\Inertia;
+use Illuminate\Http\Request;
+use App\Models\transaction;
```

Comment:
```suggestion
use App\Models\Transaction;
```
Also rename `app\Models\transaction.php` file as `app\Models\Transaction.php`